### PR TITLE
Skip H5Oflush when the native vol is being tested for parallel 

### PR DESCRIPTION
**Pull request by raylu-hdf**
H5Oflush isn't supported for the native vol with parallel.

### CODE REVIEW

File: vol_async_test_parallel.c [hunk 1]
```diff
@@ -65,7 +65,7 @@ test_one_dataset_io(void)
     hsize_t  stride[ONE_DATASET_IO_TEST_SPACE_RANK];
     hsize_t  count[ONE_DATASET_IO_TEST_SPACE_RANK];
     hsize_t  block[ONE_DATASET_IO_TEST_SPACE_RANK];
-    hbool_t  op_failed;
+    hbool_t  op_failed, is_native_vol;
```
Comment (derobins) [inline vol_async_test_parallel.c:68]: You should initialize is_native_vol to something (probably FALSE) before passing it as an out parameter. If the function doesn't set it, it'll be used uninitialized.

File: vol_async_test_parallel.c [hunk 2]
```diff
@@ -705,7 +710,7 @@ test_multi_file_dataset_io(void)
     hsize_t  stride[MULTI_FILE_DATASET_IO_TEST_SPACE_RANK];
     hsize_t  count[MULTI_FILE_DATASET_IO_TEST_SPACE_RANK];
     hsize_t  block[MULTI_FILE_DATASET_IO_TEST_SPACE_RANK];
-    hbool_t  op_failed;
+    hbool_t  op_failed, is_native_vol;
```
Comment (derobins) [inline vol_async_test_parallel.c:713]: Same here. Initialize before passing as an out parameter.

File: vol_async_test_parallel.c [hunk 3]
```diff
@@ -1503,7 +1513,7 @@ test_set_extent(void)
     hsize_t  stride[SET_EXTENT_TEST_SPACE_RANK];
     hsize_t  count[SET_EXTENT_TEST_SPACE_RANK];
     hsize_t  block[SET_EXTENT_TEST_SPACE_RANK];
-    hbool_t  op_failed;
+    hbool_t  op_failed, is_native_vol;
```
Comment (derobins) [inline vol_async_test_parallel.c:1516]: And here.

File: vol_async_test_parallel.c [hunk 4]
```diff
@@ -1817,10 +1832,10 @@ static int
 test_attribute_exists(void)
 {
     hsize_t *dims = NULL;
-    hbool_t  op_failed;
+    hbool_t  op_failed, is_native_vol;
```
Comment (derobins) [inline vol_async_test_parallel.c:1835]: And here, too.

**Review by jhendersonHDF — APPROVED** [review]
Other than Dana's comments about initializing `is_native_vol`, I think this looks fine. I was more inclined to just skip the entire tests, but this is probably good for testing that the async routines just pass through correctly for the native VOL in parallel.

**Comment by raylu-hdf** [comment]
This PR is on hold as Neil is considering changing the library to avoid this problem.

File: vol_async_test_parallel.c (hunk 1 above)
Comment (raylu-hdf) [reply vol_async_test_parallel.c]: Fixed.

File: vol_async_test_parallel.c (hunk 2 above)
Comment (raylu-hdf) [reply vol_async_test_parallel.c]: Done.

File: vol_async_test_parallel.c (hunk 3 above)
Comment (raylu-hdf) [reply vol_async_test_parallel.c]: Changed.

File: vol_async_test_parallel.c (hunk 4 above)
Comment (raylu-hdf) [reply vol_async_test_parallel.c]: Corrected.

**Review by derobins — APPROVED** [review]

**Pull request merged**